packages/webview_flutter/webview_flutter_wkwebview/lib/src/foundation/foundation.dart

```diff
@@ -6,6 +6,86 @@ import 'dart:typed_data';
 
 import 'package:flutter/foundation.dart';
 
+/// The values that can be returned in a change map.
+///
+/// Wraps [NSKeyValueObservingOptions](https://developer.apple.com/documentation/foundation/nskeyvalueobservingoptions?language=objc).
+enum NSKeyValueObservingOptions {
+  /// Indicates that the change map should provide the new attribute value, if applicable.
+  ///
+  /// See https://developer.apple.com/documentation/foundation/nskeyvalueobservingoptions/nskeyvalueobservingoptionnew?language=objc.
+  new_,
```

Comment:
This looks like a typo at first. How about `includeNew` and `includeOld` to avoid collisions in a less confusing way?

Reply:
Updated the enums with `newValue`, `oldValue`, `initialValue`, and `priorNotification`.